Shared/Samples/Augment reality to collect data/AugmentRealityToCollectDataView.swift

```diff
@@ -0,0 +1,210 @@
+    @State private var graphicsOverlay: GraphicsOverlay = {
+        let graphicsOverlay = GraphicsOverlay()
+        let tappedPointSymbol = SimpleMarkerSceneSymbol(
+            style: .diamond,
+            color: .orange,
+            height: 0.5,
+            width: 0.5,
+            depth: 0.5,
+            anchorPosition: .center
+        )
+        graphicsOverlay.renderer = SimpleRenderer(symbol: tappedPointSymbol)
+        graphicsOverlay.sceneProperties.surfacePlacement = .absolute
+        return graphicsOverlay
+    }()
+    /// The status text displayed to the user.
+    @State private var statusText = "Tap to create a feature"
+    /// The selected tree health for the new feature.
+    @State private var treeHealth: TreeHealth?
+    /// A Boolean value indicating whether a feature can be added .
+    @State private var canAddFeature = false
+    /// A Boolean value indicating whether the tree health action sheet is presented.
+    @State private var treeHealthSheetIsPresented = false
+    /// The error shown in the error alert.
+    @State private var error: Error?
+    
+    init() {
+        let featureLayer = FeatureLayer(featureTable: featureTable)
+        featureLayer.sceneProperties.surfacePlacement = .absolute
+        scene.addOperationalLayer(featureLayer)
+    }
+    
+    var body: some View {
+        if #available(iOS 16, *) {
+            NavigationStack {
+                augmentRealityToCollectDataView
+            }
+        } else {
+            NavigationView {
+                augmentRealityToCollectDataView
+            }
+        }
+    }
+    
+    @MainActor @ViewBuilder var augmentRealityToCollectDataView: some View {
+        VStack(spacing: 0) {
+            WorldScaleSceneView { _ in
+                SceneView(scene: scene, graphicsOverlays: [graphicsOverlay])
+            }
+            .calibrationButtonAlignment(.bottomLeading)
+            .onCalibratingChanged { isPresented in
+                scene.baseSurface.opacity = isPresented ? 0.5 : 0
+            }
+            .onSingleTapGesture { _, scenePoint in
+                graphicsOverlay.removeAllGraphics()
+                canAddFeature = true
+                
+                // Add feature graphic.
+                graphicsOverlay.addGraphic(Graphic(geometry: scenePoint))
+                statusText = "Placed relative to ARKit plane"
+            }
+            .task {
+                do {
+                    try await featureTable.load()
+                } catch {
+                    self.error = error
+                }
+            }
+            .overlay(alignment: .top) {
+                Text(statusText)
+                    .multilineTextAlignment(.center)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+            Divider()
```

Comment:
What is this divider for? It seems like it shouldn't be necessary, which seems like the V stack shouldn't be necessary. Am I missing something?

Reply:
Wow, that is…unexpected.